backend/src/test/java/com/github/chuettenrauch/mixifyapi/unit/mixtape/controller/MixtapeControllerTest.java

```diff
@@ -0,0 +1,121 @@
+package com.github.chuettenrauch.mixifyapi.unit.mixtape.controller;
+
+import com.github.chuettenrauch.mixifyapi.mixtape.controller.MixtapeController;
+import com.github.chuettenrauch.mixifyapi.mixtape.service.MixtapeService;
+import com.github.chuettenrauch.mixifyapi.security.config.SecurityConfig;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.ImportAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Named.named;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.oauth2Login;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(MixtapeController.class)
+@ImportAutoConfiguration(classes = SecurityConfig.class)
+class MixtapeControllerTest {
+
+    @Autowired
+    private MockMvc mvc;
+
+    @MockBean
+    private MixtapeService mixtapeService;
+
+    @Test
+    void create_whenNotLoggedIn_thenReturnUnauthorized() throws Exception {
+        this.mvc.perform(post("/api/mixtapes"))
+                .andExpect(status().isUnauthorized());
+    }
+
```

Comment:
Du hast alle Test die unauthorized zurückgegeben hätten aus dem TrackController rausgehauen, ich würde gern wissen warum du die hier drin gelassen hast?

Reply:
Hab ich glaub ich einfach vergessen - zieh ich raus

Reply:
Ach so moment. Das is der WebMvcTest - die hab ich da ja hin umgezogen